platform/smallstep-api.mdx

```diff
@@ -71,43 +62,46 @@ Output:
 ]
 ```
 
-### Configure Devices and Workloads via the API
-
-Here’s the most common workflow for using the Smallstepi API, either directly or via the Terraform or Ansible integrations:
-
-#### 1. [Create a device collection](https://gateway.smallstep.com/operations/PutDeviceCollection).
-    
-Set the device type according to your deployment. Note that each device type is going to have its own `deviceTypeConfiguration` as well. For example, an Azure VM device will require you to set your Azure tenant ID, for access control.
-    
-You’ll register your individual devices in step 3.
-    
-#### 2. [Add workloads](https://gateway.smallstep.com/operations/PutWorkload).
-    
-This is where you set up your workload configuration. You can specify things like certificate SANs and duration, certificate and key file ownership and permissions, reload options for the workload, and so on.
-
-A single workload can be associated with multiple devices.
-
-Have three HAProxy hosts in front of a fleet of `internal.example.com` web servers? That’s one workload.
-
-Have a PostgreSQL server with several hot standby servers? That’s one workload.
-
-Have an etcd cluster comprised of three hosts? That’s one workload.
-
-On this endpoint, you can use the `deviceMetadataKeySANs` property to dynamically map certificate SANs for each device that’s running a workload. Or, statically assign SANs for every certificate in the workload, using `staticSANs`.
-    
-#### 3. [Add devices to collection](https://gateway.smallstep.com/operations/PutCollectionInstance).
-     
-Each device’s `instanceID` should be set according to the device type:
- 
-- For AWS EC2 instances, use the EC2 instance ID (eg. [`i-0d460b88a96dfdd08`](https://us-east-2.console.aws.amazon.com/ec2/home?region=us-east-2#InstanceDetails:instanceId=i-0d460b88a96dfdd08))
-- For Azure VMs, use the managed identity principal object ID
-- For Compute Engine VMs, use the instance ID (eg. `16979701088048819`)
- 
-Use the `data` field to store any relevant instance metadata, such as:
- 
-- an instance’s region, name, or role(s)
-- DNS names (these can be mapped to a SANs using `deviceMetadataKeySANs`, above)
-
-#### 4. Install the Smallstep app on each device.
-    
-See [Smallstep App](./smallstep-app.mdx) for details.
+### Example: Add devices via the API
+
+You can import devices from any source into Smallstep using our API.
+
+Devices added via API are automatically approved.
+but they will not be marked as high-assurance
+until Smallstep receives an attestation from the device.
+
+#### Example: I have a list of device identifiers
+
+For each device, use the [Save Collection Instance](https://gateway.smallstep.com/v2023-11-01/operations/PutCollectionInstance) endpoint to create a device.
```

Comment:
We have a [new endpoint in the v2025-01-01 API version for adding devices](https://gateway.smallstep.com/v2025-01-01/operations/PostDevices). The version is incomplete - there's no support for adding accounts or listing devices - so we don't link to it yet.

Reply:
@areed I don't understand. Are you saying we should wait for that piece to be more complete, merge this as-is, or update the instructions here to match the new endpoint?

Reply:
I'd say merge as-is